.travis.yml

```diff
@@ -12,7 +12,8 @@ install:
 - npm install
 - bower install
 after_success:
-- sed -i -- 's/SF:ember-frost-sidebar\/\(.*\)/SF:addon\/\1.js/' coverage/lcov.info && rm -f coverage/lcov.info--
```

Comment:
Excluding the 'secure' keys generated here, the rest of this file should have the same content as https://github.com/ciena-frost/ember-frost-sort/blob/master/.travis.yml